code/datums/components/harvester.dm

```diff
@@ -250,7 +250,7 @@
 
 ///Handles behavior when attacking a mob with bicaridine
 /datum/component/harvester/proc/attack_bicaridine(datum/source, mob/living/target, mob/living/user, obj/item/weapon)
-	if(isxeno(target)) //Self-heal on xeno strike
+	if(target.faction != user.faction) //Self-heal on attacking an enemy
```

Comment:
IFF sord is lame.

Reply:
What would be the preferred fix? Purely just an intent check?

Reply:
yes